### DISCUSSION TITLE
I can't clear DRM from epub

### DISCUSSION THREAD

**Discussion by Coderonline1** [apprenticeharper/DeDRM_tools#2145] [General]
Hello,

I am trying convert a .epub (from ADE) with DRM through this articles:

- https://robin-horton.medium.com/how-to-convert-adobe-digital-editions-to-pdf-with-calibre-e5ce75887fe5
- https://robin-horton.medium.com/how-to-install-a-plugin-to-calibre-cadb57bf7765

I just downgrade the version of calibre but the behavior remains the same. Thr output on version calibre 6.5 is:

`ApplicationPaletteChange event ignored
Using calibre Qt style: True
ApplicationPaletteChange event received
calibre Debug log
calibre 6.5  embedded-python: True
Windows-10-10.0.19044-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19044')
Python 3.10.1
Windows: ('10', '10.0.19044', 'SP0', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: DeDRM (7, 2, 1)
calibre 6.5  embedded-python: True
Windows-10-10.0.19044-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19044')
Python 3.10.1
Windows: ('10', '10.0.19044', 'SP0', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: DeDRM (7, 2, 1)
QPA platform: windows
devicePixelRatio: 1.0
logicalDpi: 96.0 x 96.0
physicalDpi: 96.0 x 95.98837209302326
[0.00] Starting up...
[0.00] Showing splash screen...
[0.19] splash screen shown
[0.19] Initializing db...
[0.19] db initialized
[0.19] Constructing main UI...
[1.05] main UI initialized...
[1.05] Hiding splash screen
Starting QuickView
DeDRM v7.2.1: Trying to decrypt dsfdsf.epub
DeDRM v7.2.1: Verifying zip archive integrity
DeDRM v7.2.1: dsfdsf.epub is a secure Adobe Adept ePub
DeDRM v7.2.1: Looking for new default Adobe Digital Editions Keys after 0.3 seconds
DeDRM v7.2.1: Exception when getting default Adobe Key after 0.3 seconds
Traceback (most recent call last):
  File "calibre_plugins.dedrm.adobekey", line 358, in adeptkeys
FileNotFoundError: [WinError 2] The system cannot find the file specified

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "calibre_plugins.dedrm.__init__", line 358, in ePubDecrypt
  File "calibre_plugins.dedrm.adobekey", line 361, in adeptkeys
calibre_plugins.dedrm.adobekey.ADEPTError: Adobe Digital Editions not activated
DeDRM v7.2.1: Ultimately failed to decrypt after 0.3 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "calibre\customize\ui.py", line 178, in _run_filetype_plugins
  File "calibre_plugins.dedrm.__init__", line 644, in run
  File "calibre_plugins.dedrm.__init__", line 415, in ePubDecrypt
calibre_plugins.dedrm.DeDRMError: DeDRM v7.2.1: Ultimately failed to decrypt after 0.3 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Exception ignored in: <function NativeFunction.__del__ at 0x00000282DDFB6CB0>
Traceback (most recent call last):
  File "calibre_plugins.dedrm.adobekey", line 268, in __del__
TypeError: 'NoneType' object is not callable
Exception ignored in: <function NativeFunction.__del__ at 0x00000282DDFB6CB0>
Traceback (most recent call last):
  File "calibre_plugins.dedrm.adobekey", line 268, in __del__
TypeError: 'NoneType' object is not callable
[12.09] splash screen hidden
[12.09] Started up in 12.09 seconds with 1 books
stdout+stderr from file dialog helper: [b'', b'']
piped data from file dialog helper: [b'\xdb_\x8b}\xd9h\xae\xe0.\x98":\xa1\xadg\x9e\x892H\x08\x04<y\xce \xb3\x81T#1@ ', b'Z:\\dsfdsf.epub']
Added Name of the Book to db in: 0.1
Added 1 books in 0.9 seconds
`

**Comment by ElleKayEm** [comment]
For calibre 6, use DeDRM 10.0.3: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.3

Is Adobe Digital Editions authorized with your Adobe ID and on the same PC and user as calibre with DeDRM?

**Reply by Coderonline1** [reply]
This is my error. I didn't do all job on the same computer. Thank you
